### ISSUE TITLE
推送过来的文件或者图片无法正常打开

### ISSUE THREAD

**Issue by erma2818** [chanify/chanify#146]
服务端是用群晖搭建的，正常启动后可以通过终端推送文本信息、浏览器插件推送文本信息、链接，但是无法正常推送图片或者文件，会一直显示在加载或者在0.01%下载中。

**Comment by wizjin** [comment]
可以看一下手机是不是能访问到服务端，应该是下载不到图片的原因。

**Comment by erma2818** [comment]
是的。我把配置文件中的内网链接改成内网穿透的链接就可以了。但是还有一个问题，windows上安装了，却一直推送不了信息，已经安装教程在windows同一文件夹放了配置ini，同时在群晖配置文件中加入了客户端的信息。
